### PR TITLE
[2] Clean up the codebase from the uniswap overrides

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Clean up the codebase from the uniswap overrides.

Removed:
 - `custom/utils`
 - `custom/theme`
 - `custom/hooks`
 - `custom/state`

Also, all `custom`-related things are removed from `tsconfig.json` and from `craco` config

## What changed
1. Unused files are removed
2. Files from `src/custom/*` are moved to `src/*`

## How `custom` files were moved to `src`
There are two cases:
1. When `moduleMod.tsx` contains `export * from '@src/module.tsx'`
2. When it doesn't

The second case is the simplest. In this case, I just moved files from `src/custom/module` to `src/module`.
In the first case, I manually merged the original and `custom` files.

# To Test
1. Regression testing (whole app is affected)

### CODE REVIEW

File: src/state/user/hooks.tsx [hunk 1]
```diff
@@ -401,3 +397,74 @@ export function useTrackedTokenPairs(): [Token, Token][] {
...
+  const dispatch = useAppDispatch()
+
+  return useMemo(() => {
+    if (chainId) {
+      dispatch(initFavouriteTokens({ chainId }))
+    }
+  }, [chainId, dispatch])
+}
+
+export function serializeToken(token: Currency): SerializedToken {
```
Comment (alfetopito) [inline src/state/user/hooks.tsx:460]: Repeated fn with the same name and similar implementation.

See src/state/user/reducer

File: src/state/user/reducer.ts [hunk 2]
```diff
@@ -61,12 +67,50 @@ export interface UserState {
...
+      [address: string]: SerializedToken
+    }
+  }
 }
 
 function pairKey(token0Address: string, token1Address: string) {
   return `${token0Address};${token1Address}`
 }
 
+function serializeToken(token: Token): SerializedToken {
```
Comment (alfetopito) [inline src/state/user/reducer.ts:83]: Repeated fn with the same name and similar implementation.

See src/state/user/hooks

File: src/state/user/hooks.tsx (hunk 1 above)
Comment (shoom3301) [reply src/state/user/hooks.tsx:460]: The implementation is a little different:
image

image

File: package.json [hunk 3]
```diff
@@ -8,14 +8,14 @@
...
     "build": "yarn i18n:compile && craco build && yarn writeVersion",
-    "test": "NODE_PATH=src/custom craco test --env=jsdom",
+    "test": "craco test --env=jsdom",
     "start:ssl": "HTTPS=true yarn start",
     "start:service-worker": "yarn build && yarn serve",
     "mock": "REACT_APP_MOCK=true yarn start",
     "build:analyze": "cross-env REACT_APP_ANALYZE_BUNDLE=true yarn build",
     "ipfs:build": "cross-env PUBLIC_URL=\".\" yarn build",
     "ipfs:publish": "ipfs-deploy build -p pinata -O",
-    "test:debug": "NODE_PATH=src/custom craco --inspect-brk test --runInBand --no-cache",
```
Comment (anxolin) [inline package.json:18]: no need to change, but we might not need craco soon, it was introduced just to do the overlay layer on top of Uni

**Review by anxolin — APPROVED** [review]

File: src/state/user/hooks.tsx (hunk 1 above)
Comment (shoom3301) [reply src/state/user/hooks.tsx:460]: Added TODO, thanks!

**Pull request merged**